### ISSUE TITLE
Add fields to enter number for each packing type and storage accessibility chosen

### ISSUE THREAD

**Issue by HannahCastro** [vasavah/vsv-fli-grn#228]
**Is your feature request related to a problem? Please describe.**
Since data for packing type and storage accessibility for storage location are already available. It will be better if we can indicate how many of each option is allowed in the storage location.
Example: For packing type, bin and box were chosen. Then, how many bins and boxes can be stored in the location. Same for storage accessibility, how many trucks of a certain size can enter at 1 time?

**Describe the solution you'd like**
For each packing type and storage accessibility chosen, there should be a corresponding field to enter the number. For example, if 3 packing types were chosen, then there will be 3 fields to enter the number. Then if 2 storage accessibility options were chosen, then 2 fields will be shown to add the number as well.

**Additional context**
![image](https://user-images.githubusercontent.com/51902789/81881310-75c10700-95c2-11ea-8e9c-c7424748a6ed.png)
